src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -111,19 +128,21 @@ export const TxDetails = ({ transaction }: TxDetailsProps): ReactElement => {
   }
 
   return (
-    <TxDetailsContainer>
+    <TxDetailsContainer ownerRows={hasModule ? 3 : 2}>
       <div className={cn('tx-summary', { 'will-be-replaced': willBeReplaced })}>
         <TxSummary txDetails={data} />
       </div>
+      {isMultiSend && getModuleDetails()}
```

Comment:
Why are we calling the `ModuleDetails()` if it is a `multisend` ? I think it returns always in https://github.com/gnosis/safe-react/blob/a03c0765663ebf5b4eb36403af023b079eaea201/src/routes/safe/components/Transactions/TxList/TxDetails.tsx#L101-L103

Reply:
The positioning of the module needs to be above the mapped MultiSend details. It has to be in this component to avoid prop drilling.